core/src/cluster_nodes.rs

```diff
@@ -39,7 +43,8 @@ enum NodeId {
     Pubkey(Pubkey),
 }
 
-struct Node {
+// pub for bench
```

Comment:
If someone removes `pub`, the code will not compile; so the comment will be redundant.
Similarly other places below.